scripts/build_airocean_parameters.py

```diff
@@ -0,0 +1,560 @@
+    Definition of the airocean "base" (read non-split) faces and centers
+"""
+
+airocean_face_vertices = airocean_vertices[
+    [
+        [12, 16, 14],  # Face 0
+        [12, 13, 16],  # Face 1
+        [12, 9, 13],  # Face 2
+        [12, 8, 9],  # Face 3
+        [12, 14, 10],  # Face 4
+        [14, 16, 18],  # Face 5
+        [20, 16, 17],  # Face 6
+        [17, 16, 13],  # Face 7
+        [9, 11, 13],  # Face 9
+        [9, 5, 7],  # Face 10
+        [9, 8, 5],  # Face 11
+        [5, 8, 4],  # Face 12
+        [4, 8, 6],  # Face 13
+        [0, 4, 2],  # Face 14
+        [19, 17, 15],  # Face 16
+        [3, 7, 5],  # Face 17
+        [1, 5, 4],  # Face 18
+        [1, 4, 0],  # Face 19
+        [15, 17, 13],  # Face 8
+        [19, 21, 17],  # Face 15
+        [26, 1, 0],  # Face 8'
+        [11, 15, 13],  # Face 15'
+    ]
+]
+airocean_face_vertices = np.concatenate(
+    [
+        airocean_face_vertices,
+        np.zeros((*airocean_face_vertices.shape[:-1], 1)),
+    ],
+    axis=2,
+)
+
+airocean_22_centers = airocean_face_vertices.mean(axis=1)
+airocean_22_centers[:, 2] = 1
+airocean_2_centers = airocean_22_centers[[18, 19]]
+
+
+"""
+    Definition of the complete faces of the
+    airocean unfolded net (23 faces total).
+"""
+
+airocean_23_face_vertices = airocean_vertices[
+    [
+        [12, 16, 14],  # Face 0
+        [12, 13, 16],  # Face 1
+        [12, 9, 13],  # Face 2
+        [12, 8, 9],  # Face 3
+        [12, 14, 10],  # Face 4
+        [14, 16, 18],  # Face 5
+        [20, 16, 17],  # Face 6
+        [17, 16, 13],  # Face 7
+        # [15, 17, 13],  # Face 8, split
+        [9, 11, 13],  # Face 9
+        [9, 5, 7],  # Face 10
+        [9, 8, 5],  # Face 11
+        [5, 8, 4],  # Face 12
+        [4, 8, 6],  # Face 13
+        [0, 4, 2],  # Face 14
+        # [19, 21, 17],  # Face 15, split
+        [19, 17, 15],  # Face 16
+        [3, 7, 5],  # Face 17
+        [1, 5, 4],  # Face 18
+        [1, 4, 0],  # Face 19
+        [17, 22, 21],  # Face 20 - from 15 (Ocean)
+        [1, 0, 23],  # Face 21 - from 15 (Australia)
+        [15, 17, 24],  # Face 22 - from 8 (Japan)
+        [17, 13, 24],  # Face 23 - from 8 (Japan)
+        [11, 25, 13],  # Face 24 - from 8 (Japan)
+    ]
+]
+airocean_23_face_vertices = np.concatenate(
+    [
+        airocean_23_face_vertices,
+        np.ones((*airocean_23_face_vertices.shape[:-1], 1)),
+    ],
+    axis=2,
+)
+airocean_23_face_vertices[:, :, 3] = 1.0
+
+
+airocean_23_centers = airocean_23_face_vertices.mean(axis=1)
+airocean_23_centers[:, 2] = 1
+
+
+"""
+    Definition of the forward and inverse transform matrices to switch
+    from vertical to horizontal layout
+"""
+
+
+def translation(dx=0, dy=0):
+    res = np.eye(4)
+    res[[0, 1], 3] = dx, dy
+    return res
+
+
+def rotation(angle):
+    res = np.eye(4)
+    res[:2, :2] = np.round(
+        [
+            [np.cos(angle), -np.sin(angle)],
+            [np.sin(angle), np.cos(angle)],
+        ],
+        10,
+    )
```

Comment:
keep full precision, but remove the near zeros from `np.cos(np.pi / 2)`
```suggestion
    res[:2, :2] = zap_zero_or_one(
        np.array(
            [
                [np.cos(angle), -np.sin(angle)],
                [np.sin(angle), np.cos(angle)],
            ]
        )
    )
```